apps/web/tailwind.config.js

```diff
@@ -19,15 +18,24 @@ module.exports = {
       'venice-black-drop-shadow': `0px 2px 4px 0px ${VeniceTheme.dropShadow}`,
     },
     extend: {
-      backgroundColor: {
-        'venice-black': VeniceTheme.black,
+      colors: {
+        ...VeniceTheme,
+        'venice-black': {
+          DEFAULT: VeniceTheme.black,
+          300: '#4e4e4e',
+          500: '#2e2e2e',
+        },
```

Comment:
Need to provide the mapping to the theme so we can do `bg-venice-black-300` or `bg-venice-black-500`. `bg-venice-black` will use the `DEFAULT` i.e. maintains the same behavior.

We'll add more overtime as we need it.